src/hooks/useSortedYearnVaults/useSortedYearnVaults.ts

```diff
@@ -0,0 +1,84 @@
+import { SupportedYearnVault } from 'features/defi/providers/yearn/api/vaults'
```

Comment:
Hey chxash, this looks great. Today we merged in a change that will effect your PR. We moved out the yearn api stuff into it's own package.

It shouldn't effect your PR too much, but you'll want to merge in the latest master, and instead of importing from features/defi/... you'll import from @shapeshiftoss/investor-yearn

```suggestion
import { SupportedYearnVault } from '@shapeshiftoss/investor-yearn'
```

Reply:
Done ! updated with latest develop